### PR TITLE
openstack-crowbar: fixes for maintenance update

### PR DESCRIPTION
**Pull request by flaviodsr**
This change include several fixes for issues identified on maintenance update jobs:

- The setup_zypper_repos role should be called on the pre_tasks section of
the playbook or the handler that refresh the repositories will be
triggered only after the update.
- The tempest barclamp relies on the service catalog to identify when a
service is enabled. If the tempest barclamp is applied before any other
service (e.g. manila) tempest will be configured to skip that service
(e.g. manila = false) as it will not be on the service catalog yet.
- The "Check MU URL" task occasionally fails probably due to temporally
network issues. This change adds a retry to that task and also increases
its connection timeout from 30 to 60 seconds.
- On SES deployment: Due to temporary network issues adding the base 
repositories occasionally fails (e.g. "Error message: Could not resolve host:
provo-clouddata.cloud.suse.de). This change adds retries to that task.
- To ensure that the update changes are in place and it did not break
anything, run chef-client on each cloud node before reapplying the
barclamps.

### CODE REVIEW

**Comment by flaviodsr** [comment]
Tested in:
- cloud9: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar9-job-mu-ha-update-x86_64/detail/cloud-crowbar9-job-mu-ha-update-x86_64/75/pipeline
- cloud8: https://ci.suse.de/blue/organizations/jenkins/cloud-crowbar8-job-mu-ha-update-x86_64/detail/cloud-crowbar8-job-mu-ha-update-x86_64/63/pipeline

File: scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+crowbar_proposal_name:
+  pacemaker: "services"
```
Comment (stefannica) [inline scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml]: You need to get the actual list of pacemaker proposals. This only works if you have one cluster, but remember that we also support std-split and other scenarios that split the control plane into several clusters.

Comment (flaviodsr) [reply scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml]: Is this just for pacemaker or other barclamps can also have other proposals besides the `default` ?

Comment (stefannica) [reply scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml]: no, pacemaker is "special" that way

Comment (flaviodsr) [reply scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml]: alright, will make that change. thanks!

Comment (flaviodsr) [reply scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml]: Turns out that reapplying the barclamp proposals are not the right procedure to update. This commit was dropped from that PR and will be made in a following PR.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -6041,6 +6041,9 @@ function onadmin_batch
         get_novacontroller
         safely oncontroller magnum_service_setup
     fi
+    if grep -q "barclamp: tempest" ${scenario}; then
+        safely crowbar batch --include tempest --timeout 2400 build ${scenario}
```
Comment (rsalevsky) [inline scripts/qa_crowbarsetup.sh]: Why having a lower default timeout here?

Comment (flaviodsr) [reply scripts/qa_crowbarsetup.sh]: right, I forgot that those timeouts were removed. thanks

**Pull request merged**